TileFormats/Batched3DModel/README.md

```diff
@@ -49,86 +68,93 @@ in the Cesium implementation of 3D Tiles.
-
-The binary glTF immediately follows the feature table and batch table.  It may embed all of its geometry, texture, and animations, or it may refer to external sources for some or all of these data.
+Batched 3D Model uses [glTF 2.0](https://github.com/KhronosGroup/glTF/tree/master/specification/2.0) to embed model data.
 
-The glTF asset must be 8-byte aligned so that glTF's byte-alignment guarantees are met. This can be done by padding the Feature Table or Batch Table if they are present.
+The [binary glTF](https://github.com/KhronosGroup/glTF/tree/master/specification/2.0#binary-gltf-layout) immediately follows the Feature Table and Batch Table.  It may embed all of its geometry, texture, and animations, or it may refer to external sources for some or all of these data.
 
 As described above, each vertex has a `batchId` attribute indicating the model to which it belongs.  For example, vertices for a batch with three models may look like this:
 ```
 batchId:  [0,   0,   0,   ..., 1,   1,   1,   ..., 2,   2,   2,   ...]
 position: [xyz, xyz, xyz, ..., xyz, xyz, xyz, ..., xyz, xyz, xyz, ...]
 normal:   [xyz, xyz, xyz, ..., xyz, xyz, xyz, ..., xyz, xyz, xyz, ...]
 ```
-Vertices do not need to be ordered by `batchId` so the following is also OK:
+Vertices do not need to be ordered by `batchId`, so the following is also OK:
 ```
 batchId:  [0,   1,   2,   ..., 2,   1,   0,   ..., 1,   2,   0,   ...]
 position: [xyz, xyz, xyz, ..., xyz, xyz, xyz, ..., xyz, xyz, xyz, ...]
 normal:   [xyz, xyz, xyz, ..., xyz, xyz, xyz, ..., xyz, xyz, xyz, ...]
 ```
 Note that a vertex can't belong to more than one model; in that case, the vertex needs to be duplicated so the `batchId`s can be assigned.
 
-The `batchId` is identified by the glTF technique parameter semantic `_BATCHID`.  For example:
+The `batchId` parameter is specified in a glTF mesh [primitive](https://github.com/KhronosGroup/glTF/tree/master/specification/2.0#reference-primitive) by providing the `_BATCHID` attribute semantic, along with the index of the `batchId` [accessor](https://github.com/KhronosGroup/glTF/tree/master/specification/2.0#accessors). For example, 
 
 ```JSON
-"technique": {
-    "attributes": {
-        "a_batchId": "batchId"
-    },
-    "parameters": {
-        "batchId": {
-            "semantic": "_BATCHID",
-            "type": 5126
+"primitives": [
+    {
+        "attributes": {
+            "_BATCHID": 0
         }
     }
-}
+]
 ```
 
-For this example the attribute is named `a_batchId`, and is declared in the vertex shader as:
-
-```glsl
-attribute float a_batchId;
+```JSON
+{
+    "accessors": [
+        {
+            "bufferView": 1,
+            "byteOffset": 0,
+            "componentType": 5125,
+            "count": 4860,
+            "max": [2],
+            "min": [0],
+            "type": "SCALAR"
```

Comment:
Below this example mention that `accessor.type` must be `"SCALAR"`, but there are no restrictions on the other properties (as long as they are in accordance with glTF).